client-src/runtar.c

```diff
@@ -192,8 +192,9 @@ main(
 		g_str_has_prefix(argv[i],"--exclude-from") ||
 		g_str_has_prefix(argv[i],"--files-from")) {
```

Comment:
Hello, would this work?
```c
		if (strchr(argv[i], '=')) {
		    good_option++;
		} else {
		    /* Accept theses options with the following argument */
		    good_option += 2;
		}
```

Reply:
Hi,

yeah, i agree that this is a more elegant solution, would have been nice if this was suggested before code got merged. maybe a PR in lines of enhancement to argument checking in runtar ?

Reply:
Well there was not much time for review of the PR. I will send you a separate PR.

Reply:
Sure, Thanks for your inputs.

Reply:
PR #206 against master branch (which does not have this fix yet, as your PR is for the 3_5 branch).

I can submit a PR against 3_5 as well.

Reply:
Thanks for the PR, i don't think a separate PR would be necessary for `3_5` as we can pick the changes from master.